pull.sh

```diff
@@ -9,3 +9,10 @@ rm smartmontools_$VERSION-$REVISION.debian.tar.xz
 wget http://deb.debian.org/debian/pool/main/s/smartmontools/smartmontools_$VERSION.orig.tar.xz
 tar xf smartmontools_$VERSION.orig.tar.xz --strip 1
 rm smartmontools_$VERSION.orig.tar.xz
+
+cp NAS-131181.patch debian/patches
+cp NAS-131327.patch debian/patches
+echo 'NAS-131181.patch' >> debian/patches/series
+echo 'NAS-131327.patch' >> debian/patches/series
+
+echo -e "$(cat changelog)\n\n$(cat debian/changelog)" > debian/changelog
```

Comment:
Are the cat's inside echo a good idea?  If their results are passed to echo command line, then I wonder about command line size limit, escaping, etc.

Reply:
Thanks @amotin. Escaping could indeed be an issue, I have updated it in my latest push.

Reply:
Thanks. But now I wonder if you need `mv -f` to overwrite the original file?

Reply:
Hmm, it still overwrote without `-f` but I added `-f`.